### ISSUE TITLE
ebs_snapshot_count seems high

### ISSUE THREAD

**Issue by cariaso** [brennerm/aws-quota-checker#2]
I'm seeing
EBS Snapshots per region [default/eu-west-1]: 25780/100000 ✓
for an account with only a handful of actual snapshots. 
Perhaps change
```
return len(self.boto_session.client('ec2').describe_snapshots()['Snapshots'])
```

To
```
return len(self.boto_session.client('ec2').describe_snapshots(OwnerIds=["self"])['Snapshots'])
```

**Comment by brennerm** [comment]
Hey @cariaso, thanks for reporting. I was also wondering about this high number but I couldn't find an info whether this limit only applies to self owned snapshots. It's quite likely but if not it would also explain why the limit is so high.

Let me contact AWS support and try to clarify that.

**Comment by brennerm** [comment]
Verified that only self owned snaphots count towards this quota. Fixed with #16.
